README.md

```diff
@@ -99,10 +99,11 @@ jenkins:
     (...)
 
   nodes:
-    slave:
-      name: "static-slave"
-      remoteFS: "/home/jenkins"
-      launcher: "jnlp"
+    - permanent:
+        name: "static-slave"
+        remoteFS: "/home/jenkins"
+        launcher:
```

Comment:
To be honest, I failed to understand that this is a short notation 😬
At the beginning I faced a couple of errors on different pieces of my CASC.yaml at once. 
While looking for a solution, I wondered about the differnt notations for workers (`slave` & `dumb`). I then decided for `dumb`, because 
* a list seemed more intuitive to me to have to possibility to define multiple workers
* I thought the term `slave` is outdated as of it's political incorrectness 😉

Finally, I got an error or a warning for `dumb` and then ended up with `permantent` which worked fine.

Quite a story, huh?

I think it would be worth documenting the different notations in the `build_agents` demo.

Reply:
the symbol for `hudson.model.Slave` is indeed a complex one, fixed for casc in recent jenkins-core release.